test/kitchen/test/integration/dd-agent-install-script/rspec/dd-agent-install-script_spec.rb

```diff
@@ -2,4 +2,26 @@
 
 describe 'dd-agent-installation-script' do
   include_examples 'Agent'
+
+  context 'when testing the install infos' do
+    let(:install_info_path) do
+      if os == :windows
+        conf_path = "#{ENV['ProgramData']}\\Datadog\\install_info"
+      else
```

Comment:
You can probably remove the Windows part here, there's no install script on Windows.

Reply:
Yeah sure!